.github/release-please-config.json

```diff
@@ -1,10 +1,20 @@
 {
-  "$schema": "https://raw.githubusercontent.com/googleapis/release-please/refs/heads/main/schemas/config.json",
+  "packages": {
+    ".": {
+      "changelog-path": "CHANGELOG.md",
+      "release-type": "python",
+      "bump-minor-pre-major": false,
+      "bump-patch-for-minor-pre-major": false,
+      "draft": false,
+      "prerelease": false
```

Comment:
Pretty sure everything except `release-type` isn't required / set by default but if the bootstrap command added it in then I think it's fine to just leave it in.